mu-plugins/central-hub/tests/phpunit/unit/config-store/_loadConfigFromFilesystem.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ *  Tests for _load_config_from_filesystem
+ *
+ * @package    spiralWebDb\centralHub\Tests\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\ConfigStore;
+
+use function KnowTheCode\ConfigStore\_load_config_from_filesystem;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_LoadConfigFromFilesystem
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_LoadConfigFromFilesystem extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/internals.php';
+	}
+
+	/**
+	 *  Test should return false when $path_to_file is empty string.
+	 */
+	public function test_should_return_false_when_path_to_file_is_empty_string() {
```

Comment:
I don't think this is a valid edge case.  Why would anyone pass an empty string to this function?  

But if they did, it would throw an error.  Instead, test that an error is thrown here when a falsey is passed.

Hint:  It'll also throw an error if the path/to/file doesn't exist.

Reply:
@hellofromtonya Following our code review, this test was removed.